price-estimator/src/filter.rs

```diff
@@ -52,6 +71,13 @@ fn markets_filter() -> impl Filter<Extract = (TokenPair, QueryParameters), Error
         .and(warp::query::<QueryParameters>())
 }
 
+fn estimated_sell_amount_filter(
+) -> impl Filter<Extract = (TokenPair, f64, QueryParameters), Error = Rejection> + Copy {
+    warp::path!("api" / "v1" / "markets" / TokenPair / "estimated-sell-amount" / f64)
```

Comment:
nit: Can we use [path prefixes](https://docs.rs/warp/0.2.3/warp/macro.path.html#path-prefixes) for the common parts?

Reply:
Addressed in #1045